tests/generic/CMakeLists.txt

```diff
@@ -3,5 +3,10 @@
 # SPDX-License-Identifier: Apache-2.0
 #
 # Moritz Scherer <scheremo@iis.ee.ethz.ch>
+# Philip Wiese <wiesep@iis.ee.ethz.ch>
 
-add_subdirectory(testReturnZero)
+# Add test for host core
+add_subdirectory(host)
+
+# Add test for snitchCluster
+add_subdirectory(snitchCluster)
```

Comment:
I would not split `generic` into `host` and `snitchCluster`. The idea of `generic` was to contain ISA- and micro-architecture-indepenent tests (return zero, add some numbers). Since you're assuming snitch clusters, these changes could go into `chimera-open`?

Reply:
The idea here is that offloading to a snitch cluster can be done on `chimera-open` but also `chimera-convolve` and I wanted to avoid code duplication. In this regard, `generic` refers to tests that can be run with all targets. But I see that even though this is true atm, it is not guaranteed for the future. Do you see any other way that duplicating the code?

Reply:
Or include chimera-open in chimera-convolve tests